cmd/clusterctl/client/cluster/ownergraph.go

```diff
@@ -79,16 +72,9 @@ func GetOwnerGraph(namespace, kubeconfigPath string) (OwnerGraph, error) {
 	}
 	owners := OwnerGraph{}
 	for _, v := range graph.uidToNode {
-		// The Discovery function returns all secrets in the Cluster namespace. Ensure a Secret that is not part of the
-		// Cluster is not added to the OwnerGraph.
-		if v.identity.Kind == "Secret" {
-			clusterSecret, err := isClusterSecret(v.identity, cl)
-			if err != nil {
-				return OwnerGraph{}, err
-			}
-			if !clusterSecret {
-				continue
-			}
+		// If the node is not a tenant do not add it to the ownerGraph.
+		if len(v.tenant) == 0 {
```

Comment:
I think we can use graph.getMoveNodes to get the list of nodes already filtered with this criteria (so we can drop also this if)

In a future iteration, if necessary, we can also be explicit about which tenant we care about (one cluster, one clusterclass, probably one CRS), but for now it is ok to consider everything that should be moved because that's that is relevant in the test.